src/types/controlPlaneConfig.ts

```diff
@@ -8,7 +8,27 @@ export type DestinationDefinition = {
   Config: Record<string, unknown>;
 };
 
-export type Destination<DestinationConfig = Record<string, unknown>> = {
+export type AccountDefinition<Config = Record<string, unknown>> = {
+  Name: string;
+  Config: Config;
```

Comment:
if not used currently then skip it, Ideally this shouldn't be Record as all fields should be known

Reply:
I made it a generic. Config will be Record<string,unknown> only. Whichever destination will use this has to provide a type for the config.